IMU_CALIBRATION_AND_ALIGNMENT.s

```diff
@@ -426,11 +426,11 @@ FINETIME	INHINT			# RETURNS WITH INTERRUPT INHIBITED
 #
 # FUNCTIONAL DESCRIPTION
 #
-# THIS SECTON CONSISTS OF PRELAUNCH ALIGNMENT AND GYRO DRIFT TESTS
+# THIS SECTION CONSISTS OF PRELAUNCH ALIGNMENT AND GYRO DRIFT TESTS
 # INTEGRATED TOGETHER TO SAVE WORDS.  COMPASS IS COMPLETELY RESTART
 # PROOFED EXCEPT FOR THE FIRST 30 SECONDS OR SO.  PERFORMANCE TESTS OF
 # THE IRIGS IS RESTART PROOFED ENOUGH TO GIVE 75 PERCENT CONFIDENCE THAT
-# IF A RESTART OCCURS THE DATA WILL STILL BE GOOD.  GOOD PRACTICE TO RECYCL
+# IF A RESTART OCCURS THE DATA WILL STILL BE GOOD.  GOOD PRACTICE TO RECYCLE
```

Comment:
`RECYCL` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0432.jpg)